blocks/cards/cards.css

```diff
@@ -0,0 +1,300 @@
+.gd-plus-logo {
```

Comment:
@DriftingSands  This CSS selector is not very specific although it's only loaded for cards so if it is used across the board, you can put it in styles.css but if it's only for cards, please be more specific with the CSS selector.